check-services.php

```diff
@@ -58,12 +63,9 @@
     }
 }
 
-$sql = 'SELECT D.*,S.*,attrib_value  FROM `devices` AS D'
-       . ' INNER JOIN `services` AS S ON S.device_id = D.device_id AND D.disabled = 0 ' . $where
-       . ' LEFT JOIN `devices_attribs` as A ON D.device_id = A.device_id AND A.attrib_type = "override_icmp_disable"'
-       . ' ORDER by D.device_id DESC;';
+$services = $query->get();
```

Comment:
You did not call toArray() here. As long as the below code does not add fields, it will be fine though as models implement ArrayAccess.